packages/driver/src/cy/commands/xhr.js

```diff
@@ -471,6 +471,13 @@ module.exports = (Commands, Cypress, cy, state, config) => {
           })
         }
 
+        // look ahead to see if fixture exists
+        const fixturesRe = /^(fx:|fixture:)/
```

Comment:
@jennifer-shehane I am not sure about this regExp. This is a duplication from `server` package. Is there any "shared" place where I can put it and import here (in xhr command) and in server package?

Reply:
Not easily - since the driver code runs in the browser, and the server code runs in the Node process. Thus we are cautious about sharing code between the two packages since they can pull extra code unexpectedly